### PR TITLE
Onboard AMQ Streams operator on ROSA Classic

### PR DESCRIPTION
**Pull request by Sau1506mya**
Link to JIRA: [LPTOCPCI-998](https://issues.redhat.com/browse/LPTOCPCI-998)

### CODE REVIEW

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.14-lp-rosa-amq-streams-2-5-rosa-aws

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.14-lp-rosa-amq-streams-2-5-rosa-aws

**Review by im-konge — APPROVED** [review]

File: ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.14-lp-rosa.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","amq-streams-lp"]},
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","amq-streams-lp"]},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-lp","amq-streams-lp"]},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","amq-streams-lp"]},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
```
[the remark this answers is not in the document]
Comment (Sau1506mya) [reply ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.14-lp-rosa.yaml:37]: Hi, so this is what PQE wants. I accordingly made the changes.
Also, this value is same for `ocp4.14-lp-interop.yaml` and `ocp4.15-lp-interop.yaml` files which earlier onboarded for this product.

**Pull request merged**